src/lib/stark/createKeyPair.js

```diff
@@ -19,8 +19,9 @@ module.exports = (privateKey) => {
       x: fullPublicKey.pub.getX().toString('hex'),
       y: fullPublicKey.pub.getY().toString('hex')
     }
+    const starkPrivateKey = privateKey
 
-    return {privateKey, starkKeyPair, starkPublicKey}
+    return {privateKey, starkPrivateKey, starkKeyPair, starkPublicKey}
```

Comment:
I think the main intention here is to rename priateKey to starkPrivateKey. privateKey can be removed from return object. The module input parameter can be renamed to starkPrivateKey instead of privateKey and the module can return starkPrivateKey as well.

Reply:
sure, I didn't remove `privateKey` since wasn't sure if `privateKey` was needed by some other code and didn't have the time to verify it.

Reply:
done in 16354f876129285a008a0e4d95332cd82af54302